cmd/vclusterctl/cmd/create.go

```diff
@@ -170,7 +171,7 @@ func (cmd *CreateCmd) Run(args []string) error {
 	for _, value := range cmd.ExtraValues {
 		decodedString, err := getBase64DecodedString(value)
 		if err != nil {
-			if strings.Contains(err.Error(), "illegal base64 data at input byte 0") {
+			if strings.Contains(err.Error(), "illegal base64 data at input byte") {
```

Comment:
Instead of fatal when the base64 fails, can we always add it as a values file and only handle the case if there is no error to continue with the base64 file?

Reply:
Makes sense. Done :+1: